Library/Homebrew/test/migrator_spec.rb

```diff
@@ -0,0 +1,269 @@
+require "migrator"
+require "test/support/fixtures/testball"
+require "tab"
+require "keg"
+
+class Migrator
+  describe self do
```

Comment:
This is so that `MigratorNoOldnameError` and `MigratorNoOldpathError` are scoped inside `Migrator`.